crates/astria-conductor/src/data_availability/mod.rs

```diff
@@ -334,36 +464,110 @@ impl Reader {
             .in_current_span(),
         );
     }
+}
 
-    #[instrument(skip_all, fields(height))]
-    fn send_sequencer_subsets(
-        &self,
-        sequencer_subsets_res: Result<eyre::Result<Vec<SequencerBlockSubset>>, JoinError>,
-    ) -> eyre::Result<()> {
-        let subsets = match sequencer_subsets_res {
-            Err(e) => {
-                let error = &e as &(dyn std::error::Error + 'static);
-                warn!(error, "task processing sequencer data failed");
-                return Ok(());
+async fn find_latest_height(celestia_client: HttpClient) -> eyre::Result<Height> {
+    use celestia_client::celestia_rpc::HeaderClient;
+    Ok(celestia_client.header_network_head().await?.header.height)
+}
+
+#[instrument(skip_all, fields(height))]
```

Comment:
this probably doesn't compile right now? I don't see `height` in the function arguments

Reply:
For some reason this did compile and seems to work fine. The `fields(height)` has been removed.